src/sentry/apidocs/examples/project_examples.py

```diff
@@ -112,11 +112,11 @@
 
 
 class ProjectExamples:
-    CREATE_CLIENT_KEY = [
+    CREATE_CLIENT_KEY = UPDATE_CLIENT_KEY = GET_CLIENT_KEY = [
```

Comment:
if these are all the same do you need to create separate ones?

Reply:
No I should combine them